test/CMakeLists.txt

```diff
@@ -1,312 +1,144 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
-PROJECT(ArrayFire-Tests)
-
-# Find CUDA and OpenCL
-SET(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
-FIND_PACKAGE(CUDA QUIET)
-FIND_PACKAGE(OpenCL QUIET)
-
-OPTION(BUILD_SINGLE_TEST_FILE "Build tests in a single file" OFF)
-
-REMOVE_DEFINITIONS(-std=c++11)
-
-# If the tests are not being built at the same time as ArrayFire,
-# we need to first find the ArrayFire library
-IF(TARGET afcpu OR TARGET afcuda OR TARGET afopencl OR TARGET af)
```

Comment:
Will building examples in a separate location work still ?

Reply:
@umar456 ^

Reply:
I haven't tested this yet but it should be possible to do that. I would like to defer this to a later time.